### PR TITLE
Fixes #13933: NoSuchMethodError with datasources 4.3-1.4 + Rudder 4.3.7

### PR DESCRIPTION
**Pull request by fanf**
https://issues.rudder.io/issues/13933

### CODE REVIEW

**Comment by fanf** [comment]
Wait a minute, I'm correcting doc.

**Comment by fanf** [comment]
PR rebased

File: datasources/build.conf [hunk 1]
```diff
@@ -23,7 +23,7 @@ in <a href="/secure/administration/dataSourceManagement">the dedicated managemen
 # - x.y(.z): plugin major.minor.micro. Micro should be omitted. When omitted, z is assumed to be 0. 
 # For the build, we split the information between two properties, rudder branch and plugin version, 
 # which must be concaneted with "-" to build the plugin version. 
-plugin-branch=1.5-SNAPSHOT
+plugin-branch=1.4.1-SNAPSHOT
```
Comment (VinceMacBuche) [inline datasources/build.conf]: Why do you change this version we should do it in 1.5, we have already some fixes that are merged for 4.3-1.5

**Comment by fanf** [comment]
OK, merging this PR

**Pull request merged**